api/envoy/config/filter/network/mysql_proxy/v2/mysql_proxy.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package envoy.config.filter.network.mysql_proxy.v2;
+option go_package = "v2";
+
+import "validate/validate.proto";
+
+// [#protodoc-title: Mysql proxy]
```

Comment:
@rshriram is this code used in production currently? Should we mark it experimental?

Reply:
It is going to go into production soon. But I would mark it as experimental for sure until the parser lands and we emit all the stats we want. Right now, the stats are bit scarce (only ones we wanted to scrape)

Reply:
OK that's fine can you just add an experimental warning like we used to have for Lua?

Reply:
I will mark this as v1alpha1 so that the experimental part is clear